Zend/zend_compile.c

```diff
@@ -3876,6 +3876,22 @@ int zend_compile_func_get_args(znode *result, zend_ast_list *args) /* {{{ */
 }
 /* }}} */
 
+int zend_compile_func_array_key_exists(znode *result, zend_ast_list *args) /* {{{ */
+{
+	if (args->children != 2) {
```

Comment:
This needs to make sure that the arguments don't use unpacking. See the checks for ZEND_AST_UNPACK in some of the above functions.